src/main/kotlin/uk/gov/dluhc/printapi/service/aed/AedPdfTemplateDetailsFactory.kt

```diff
@@ -32,7 +32,7 @@ class AedPdfTemplateDetailsFactory(
         printRequest: AedPrintRequest
     ): Map<String, String> {
         return mapOf(
-            pdfTemplateProperties.welsh.placeholder.electoralRollNumber to printRequest.electoralRollNumber,
+            pdfTemplateProperties.welsh.placeholder.electoralRollNumber to printRequest.electoralRollNumber.uppercase(),
```

Comment:
👍

Reply:
To be honest I don't know that the value is case-insensitive and don't generally like to modify user provided entries.